app/Commands/Metric/UpdateMetricPointCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of Cachet.
+ *
+ * (c) Alt Three Services Limited
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace CachetHQ\Cachet\Commands\Metric;
+
+use CachetHQ\Cachet\Models\Metric;
+use CachetHQ\Cachet\Models\MetricPoint;
+
+class UpdateMetricPointCommand
+{
+    /**
+     * The metric point.
+     *
+     * @var \CachetHQ\Cachet\Models\MetricPoint
+     */
+    public $point;
+
+    /**
+     * The metric.
+     *
+     * @var \CachetHQ\Cachet\Models\Metric
+     */
+    public $metric;
+
+    /**
+     * The metric point value.
+     *
+     * @var int
+     */
+    public $value;
+
+    /**
+     * The metric point created at.
+     *
+     * @var string
+     */
+    public $createdAt;
```

Comment:
i wonder if it would be better to accept a Carbon object here, or maybe something that implements DateTimeInterface